source/includes/package-repository/_20-show.md.erb

```diff
@@ -0,0 +1,87 @@
+## Get package repository
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/repositories/0530f1a9-5282-4b69-be59-066a35076452' \
+      -u 'username:password' \
+      -H 'Accept: application/vnd.go.cd.v1+json'
```

Comment:
Just for the show call, and just display the etag here. Check get pipeline config documentation.